src/booleans.js

```diff
@@ -1,61 +1,90 @@
 function negate(a) {
   // your code here
+  return !a;
 };
 
 function both(a, b) {
   // your code here
+  return a && b;
 };
 
 function either(a, b) {
   // your code here
+  return a || b;
 };
 
 function none(a, b) {
   // your code here
+  return !(a || b);
 };
 
 function one(a, b) {
   // your code here
+  return (a && !b) || (!a && b);
 };
 
 function truthiness(a) {
   // your code here
+  return Boolean(a);
 };
 
 function isEqual(a, b) {
   // your code here
+  return a === b;
 };
 
 function isGreaterThan(a, b) {
   // your code here
+  return a > b;
 };
 
 function isLessThanOrEqualTo(a, b) {
   // your code here
+  return a <= b;
 };
 
 function isOdd(a) {
   // your code here
+  return a % 2 !==0;
 };
 
 function isEven(a) {
+  return a % 2==0;
```

Comment:
Missing  a space before and after the == operator.

I would also suggest to use === as default, because you would like it to be strictly equal, the type and the value as well.